diff --git a/node_modules/exceljs/lib/xlsx/xform/sheet/cell-xform.js b/node_modules/exceljs/lib/xlsx/xform/sheet/cell-xform.js
index 4171569..bdfe484 100644
--- a/node_modules/exceljs/lib/xlsx/xform/sheet/cell-xform.js
+++ b/node_modules/exceljs/lib/xlsx/xform/sheet/cell-xform.js
@@ -454,7 +454,7 @@ class CellXform extends BaseXform {
         break;
 
       case Enums.ValueType.Formula:
-        if (model.result !== undefined && style && utils.isDateFmt(style.numFmt)) {
+        if (typeof model.result === 'number' && style && utils.isDateFmt(style.numFmt)) {
           model.result = utils.excelToDate(model.result, options.date1904);
         }
         if (model.shareType === 'shared') {
